spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AppRegistryControllerTests.java

```diff
@@ -425,15 +425,15 @@ public void testListSingleApplication() throws Exception {
 		mockMvc.perform(get("/apps/source/time").accept(MediaType.APPLICATION_JSON)).andDo(print())
 				.andExpect(status().isOk()).andExpect(jsonPath("name", is("time")))
 				.andExpect(jsonPath("type", is("source")))
-				.andExpect(jsonPath("$.options[*]", hasSize(1)));
+				.andExpect(jsonPath("$.options[*]", hasSize(7)));
 	}
 
 	@Test
 	public void testListSingleApplicationExhaustive() throws Exception {
 		mockMvc.perform(get("/apps/source/time?exhaustive=true").accept(MediaType.APPLICATION_JSON))
 				.andExpect(status().isOk()).andExpect(jsonPath("name", is("time")))
 				.andExpect(jsonPath("type", is("source")))
-				.andExpect(jsonPath("$.options[*]", hasSize(2001)));
+				.andExpect(jsonPath("$.options[*]", hasSize(2022)));
```

Comment:
Was this test not being executed and thus when the tests were updated to use the latest time or log their parameter count had changed?

Reply:
Exactly... The more recent app has new properties (new version of boot and poller etc..)